### PR TITLE
refactor, chore(egen): Removes boilerplate, heading fixes, and other corrections from template

### PR DESCRIPTION
**Pull request by nileshspringml**
**REQUIRED:** 
- Updated template of notebook
- Additionally tested end to end notebook on python 3.9 

<br>
Updated template of notebook and tested end to end notebook
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb [hunk 1]
```diff
@@ -33,25 +34,27 @@
...
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Google Colaboratory logo\"><br> Open in Colab\n",
         "    </a>\n",
         "  </td>\n",
-        "  <td>\n",
-        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
-        "      View on GitHub\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/colab/import/https:%2F%2Fraw.githubusercontent.com%2FGoogleCloudPlatform%2Fvertex-ai-samples%2Fmain%2Fnotebooks%2Fofficial%2Fprediction%2Fpytorch_image_classification_with_prebuilt_serving_containers.ipynb\">\n",
+        "      <img width=\"32px\" src=\"https://lh3.googleusercontent.com/JmcxdQi-qOpctIvWKgPtrzZdJJK-J3sWE1RsfjZNwshCFgE_9fULcNpuXYTilIR2hjwN\" alt=\"Google Cloud Colab Enterprise logo\"><br> Open in Colab Enterprise\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb:44]: update colab enterprise logo URL:

- https://cloud.google.com/ml-engine/images/colab-enterprise-logo-32px.png

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb [hunk 2]
```diff
@@ -140,19 +143,26 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "i7EUnXsZhAGF"
+        "id": "7cc240e9c1a6"
       },
       "source": [
-        "## Installation\n",
-        "\n",
-        "Install the following packages required to execute this notebook."
+        "## Getting Started"
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb:149]: Get started

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb [hunk 3]
```diff
@@ -167,197 +177,158 @@
...
-        "\n",
-        "1. [Select or create a Google Cloud project](https://console.cloud.google.com/cloud-resource-manager). When you first create an account, you get a $300 free credit towards your compute/storage costs.\n",
-        "\n",
-        "2. [Make sure that billing is enabled for your project](https://cloud.google.com/billing/docs/how-to/modify-project).\n",
-        "\n",
-        "3. [Enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com).\n",
-        "\n",
-        "4. If you are running this notebook locally, you need to install the [Cloud SDK](https://cloud.google.com/sdk)."
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb:213]: remove "Please" and contract "it is"

- Wait until it's finished...

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb [hunk 4]
```diff
@@ -167,197 +177,158 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "region"
+        "id": "60763ee24ce0"
       },
       "source": [
-        "#### Region\n",
+        "### Set Google Cloud project information and initialize Vertex AI SDK\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb:251]: Vertex AI SDK for Python

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb [hunk 5]
```diff
@@ -167,197 +177,158 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "ef21552ccea8"
+        "id": "-EcIXiGsCePi"
       },
       "source": [
-        "**3. Colab, uncomment and run:**"
+        "**Only if your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb:296]: Delete "Only":

- If your bucket doesn't already exist

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb (hunk 1 above)
Comment (nileshspringml) [reply notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb]: Updated logo link

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb (hunk 5 above)
Comment (nileshspringml) [reply notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb]: Fixed the mentioned issue

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb (hunk 4 above)
Comment (nileshspringml) [reply notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb]: Fixed the mentioned issue

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb (hunk 2 above)
Comment (nileshspringml) [reply notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb]: Fixed the mentioned issue

**Comment by katiemn** [comment]
Is this draft pull request ready for review?

**Comment by nileshspringml** [comment]
> Is this draft pull request ready for review?

Yes, ready for PR review

File: notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb [hunk 6]
```diff
@@ -8,6 +8,7 @@
       },
       "outputs": [],
       "source": [
+        "# @title Copyright & License (click to expand)\n",
```
Comment (katiemn) [inline notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb:11]: Can you please remove this line?

Comment (nileshspringml) [reply notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb]: We removed required line

**Review by katiemn — APPROVED** [review]

**Pull request merged**